salt/metalk8s/addons/dex/deployed/chart.sls

```diff
@@ -29,6 +31,8 @@ stringData:
       https: 0.0.0.0:5556
       tlsCert: /etc/dex/tls/https/server/tls.crt
       tlsKey: /etc/dex/tls/https/server/tls.key
+    connectors:
+      {% endraw -%}{{ dex.spec.connectors | tojson }}{%- raw %}
```

Comment:
This works ?
For me you need to do 
```suggestion
    connectors: {% endraw -%}{{ dex.spec.connectors | tojson }}{%- raw %}
```
Anyway even if it's work I think it's clearer to do like I suggest

Reply:
Ok it's because of default manifest from DEX lets keep it